tests/features/sbom-explorer.feature

```diff
@@ -0,0 +1,178 @@
+    # `top section` refers to "upper" part visible above tabs selection
+    Then selection of tabs should be visible
+    And Info tab selector should be visible
+    And Packages tab selector should be visible
+    And Vulnerabilities tab selector should be visible
+    And Dependency Analytics Report tab should not be visible
+    # D.A.R tab not present at all, this test can be removed for future versions
+
+  Scenario: View SBOM Info (Metadata)
+    When User visits SBOM details Page
+    Then Info tab is selected # by default
+    When looking at content inside selected tab
+    Then SBOM name should be visible
+    And SBOM namespace should be visible
+    And either SPDX or CycloneDX Version should be visibile
+    And SBOM License should be visible
+    And SBOM Creation date should be visible
+    And SBOM Creator should be visible
+    And Package overview panel should be visible
+    # TODO: this Package panel needs clarification
+
+  Scenario: Downloading SBOM file
+    When User visits SBOM details Page
+    Then link to Download SBOM should be visible
+    When following the Download SBOM link
+    Then SBOM file should be downloaded
+    And downloaded file should match original SBOM json file
+    # TODO: clarify if it should be exact match/copy (including formatting)
+    #       or just in meaning (json keys/values)
+    #       or if it may even be just processed subset of information
+
+
+  Scenario: View list of SBOM Packages
+    When User visits SBOM details Page
+    And User selects Packages tab
+    Then SBOM name should be visible in top section # confirms its visible for all tabs
+
+    When looking at content inside selected tab
+    Then list of SBOM Packages should be visible
+    And list should be sorted alphabetically by Package name
+
+    Then Filter field should be visible
+    # TODO clarify - search using Filter field is CASE SENSITIVE in v1.2.2 - should it be expected in v2?
+    When Entering part of SBOM Package name shared by multiple packages into Filter field
+    Then list of SBOM Packages should be limited only to matching one
+    And list of SBOM Packages should be sorted alphabetically by Package name
+    When replacing Filter field input with value not matching any SBOM Package name
+    Then list of SBOM Packages should be empty # showing info about no-match-found?
+    When clearing the Filter field input
+    Then list of SBOM Packages should show original list of SBOM Packages again # not "all", still paginated
+
+    When looking at each individual Package entry
+    Then SBOM Package name should be visible
+    And SBOM Package version should be visible
+    And SBOM Package qualifiers should be visible
+
+  Scenario: View paginated list of SBOM Packages
+    Given There is ingested SBOM with more packages than fits in a page
+    When User visits SBOM details Page
+    And User selects Packages tab
+    Then Pagination of results works
```

Comment:
I hope we might need a generic scenario for pagination in all pages

Reply:
Yep, depends on actual code/implementation possibilities. So marking as resolved as not specific to this PR.